component_versions.sh

```diff
@@ -23,7 +23,7 @@
 #
 SAT_VERSION=3.19.3-20220930174308_3ef6021
```

Comment:
Shouldn't we update all these versions with the latest builds from the main branch?

Reply:
Fixed!